### PR TITLE
More functions doc

### PR DESCRIPTION
**Pull request by LiorKogan**

### CODE REVIEW

File: docs/commands/graph.query.md [hunk 1]
```diff
@@ -885,79 +885,89 @@ This section contains information on all supported functions from the Cypher que
...
+| toFloat(_expr_)             | Returns a floating point unchanged <br> Converts an integer to a floating point <br> Converts a string to a floating point or null <br> Returns null when _expr_ is null <br> Error on other types         |
+| toFloatList(_exprList_)     | Converts a list of values to a list of floating points. Each element in the list is converted using toFloatOrNull()                                                                                        |
+| toFloatOrNull(_expr_)       | Returns a floating point unchanged <br> Converts an integer to a floating point <br> Converts a string to a floating point or null <br> Returns null when _expr_ is null <br> Returns null for other types |
+| toInteger(_expr_) *         | Returns an integer unchanged <br> Converts a floating point to integer <br> Converts a string to an integer or null <br> Converts a Boolean to an integer (false to 0, true to 1) <br> Returns null when _expr_ is null <br> Error on other types         |
```
Comment (swilly22) [inline docs/commands/graph.query.md:963]: `Returns an integer unchanged` consider rephrasing,
e.g. returns `expr` unchanged if `expr` is an Integer

This suggestion applies to other functions with similar documentation.

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: docs/commands/graph.query.md [hunk 2]
```diff
@@ -885,79 +885,89 @@ This section contains information on all supported functions from the Cypher que
...
+| floor(_expr_) **          | When _expr_ is an integer: returns _expr_ <br> When _expr_ is floating point: returns a floating point equals to the greatest integer less than or equal to _expr_ <br> Returns null when _expr_ is null    |
+| log(_expr_)               | Returns the natural logarithm of a numeric value <br> Returns nan when _expr_ is negative, -inf when _expr_ is 0, and null when _expr_ is null                                                              |
+| log10(_expr_)             | Returns the base-10 logarithm of a numeric value <br> Returns nan when _expr_ is negative, -inf when _expr_ is 0, and null when _expr_ is null                                                              |
+| pow(_base_, _exponent_) * | Returns _base_ raised to the power of _exponent_ (equivalent to _base_^_exponent_) <br> Returns null when either is null                                                                                    |
+| rand(_expr_)              | Returns a random floating point in the range [0,1]                                                                                                                                                          |
```
[the remark this answers is not in the document]
Comment (LiorKogan) [reply docs/commands/graph.query.md:898]: Fixed. rand()

File: docs/commands/graph.query.md [hunk 3]
```diff
@@ -885,79 +885,89 @@ This section contains information on all supported functions from the Cypher que
...
+| round(_expr_) ** ***      | When _expr_ is an integer: returns _expr_ <br> When _expr_ is floating point: returns a floating point equals to the integer closest to _expr_ <br> Returns null when _expr_ is null                        |
```
[the remark this answers is not in the document]
Comment (LiorKogan) [reply docs/commands/graph.query.md:899]: Yes... That's what round() do.

File: docs/commands/graph.query.md (hunk 1 above)
Comment (LiorKogan) [reply docs/commands/graph.query.md:963]: `expr` can be 2+3, so "return `expr` unchanged" is actually wrong.
Changing to "When `expr` evaluates to an integer: returns its evaluation"

**Review by swilly22 — APPROVED** [review]

**Pull request merged**